src/Components/HighChartsWrapper/HighChartsWrapper.styles.ts

```diff
@@ -0,0 +1,99 @@
+import { IStyle } from '@fluentui/react';
+import {
+    IHighChartsWrapperStyleProps,
+    IHighChartsWrapperStyles
+} from './HighChartsWrapper.types';
+
+export const classPrefix = 'cb-highcharts-wrapper';
+const classNames = {
+    container: `${classPrefix}-container`,
+    shareButton: `${classPrefix}-share-button`
+};
+export const getStyles = ({
+    theme
+}: IHighChartsWrapperStyleProps): IHighChartsWrapperStyles => {
+    const commonTextStyling: Partial<IStyle> = { color: theme.palette.black };
+    const commonChartTextStyling: Partial<IStyle> = {
+        ...commonTextStyling,
+        fontSize: 12,
+        fontWeight: 'normal'
+    };
+    return {
+        container: [
+            classNames.container,
+            {
+                width: '100%',
+                height: '100%',
+                '.highcharts-credits': { display: 'none' }
+            }
+        ],
+        shareButton: [
+            classNames.shareButton,
+            {
+                color: `${theme.palette.black} !important`,
+                borderRadius: 2,
+                fontSize: 16,
+                marginLeft: 8,
+                ':hover': {
+                    background: theme.palette.neutralLighter
+                },
+                ':active': {
+                    background: theme.palette.neutralLight
+                }
+            }
+        ],
+        subComponentStyles: {
+            chart: {
+                root: {
+                    backgroundColor: 'var(--cb-color-glassy-modal)'
+                }
+            },
+            title: {
+                root: {
+                    color: theme.palette.black,
+                    fontSize: '14px',
+                    fontWeight: 'bold',
+                    lineHeight: '14px',
+                    zIndex: 0
+                }
+            },
+            xAxis: {
+                title: {
+                    ...commonTextStyling
+                },
+                label: {
+                    ...commonTextStyling
+                }
+            },
+            yAxis: {
+                title: {
+                    ...commonTextStyling
+                },
+                label: {
+                    ...commonTextStyling
+                }
+            },
+            legend: {
+                root: {
+                    ...commonChartTextStyling
+                },
+                hover: {
+                    fontWeight: 'bold',
+                    color: '#cccccc'
```

Comment:
so this is a tricky one, basically this is a color that Highcharts uses for disabled legend items. I need to set it here otherwise on hover it is still the color of the series that I set in the options while the series line turns into grey color:
![Disabled legend items](https://user-images.githubusercontent.com/45217314/196562166-025e1908-83c6-4d27-9bf0-2f06f9788859.gif)